ibft2-eks/aws-auth-cm.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: aws-auth
+  namespace: kube-system
+data:
+  mapRoles: |
+    - rolearn: <INSERT ARN HERE FROM CFN OUTPUTS>
```

Comment:
Would it be possible or desirable to have a script to get this value using the AWS-CLI?